### PR TITLE
Show bad users without logos

### PR DESCRIPTION
**Pull request by gentlecat**

### CODE REVIEW

File: metabrainz/templates/index/index.html [hunk 1]
```diff
@@ -155,9 +155,15 @@ <h1 class="text-danger glyphicon glyphicon-thumbs-down"></h1>
...
+                    {% if user.org_logo_url %}
+                      <div class="col-sm-12 logo-container">
+                        <span><img class="logo" src="{{ user.org_logo_url }}" /></span>
+                      </div>
+                    {% else %}
+                      <div class="col-sm-12 logo-container">
+                        <span class="name">{{ user.org_name }}</span>
+                      </div>
+                    {% endif %}
                   {% endfor %}
```
Comment (mayhem) [inline metabrainz/templates/index/index.html]: Got a screenshot of what the homepage looks like with the bad guy in it?

**Comment by gentlecat** [comment]
screenshot 2016-06-14 00 11 35

**Comment by mayhem** [comment]
Looking good. Can you make the "Test Inc" bigger? Can we go 50% bigger without looking dumb? Can you please put two bad companies in there and post another shot?

**Comment by gentlecat** [comment]
screenshot 2016-06-14 00 19 02

I'm not a fan of these blocks, to be honest.

**Comment by mayhem** [comment]
Great, thanks. LGTM. ⚡ ⛵ 🎎

**Comment by mayhem** [comment]
Got a better suggestion?

**Comment by gentlecat** [comment]
Yeah, have another paragraph below good supporters (keeping the way they are displayed currently) describing bad ones and showing some names in a similar way.

**Comment by mayhem** [comment]
I'd be up for trying that out. Do we get to keep the thumbs? :)

**Pull request merged**